lib/travis/build/script/go.rb

```diff
@@ -103,6 +108,13 @@ def uses_make
             '-f GNUmakefile || -f makefile || -f Makefile || -f BSDmakefile'
           end
 
+	  def uses_15_vendoring
+            if (go_version < '1.5' || go_version == 'go1')
```

Comment:
This isn't a bug _yet_, but it will be once there's a version 1.10 (I don't know Go's plans for versioning that far into the future but this is waiting to blow up in a few years.)

```ruby
'1.10' < '1.5'
# => true
```

May want to write a version parsing function?